cheatsheets/Swift.rb

```diff
@@ -78,7 +95,7 @@ class MyClass {
             }
             didSet { // never set greater than 10
```

Comment:
For clarity I'd update the comment to reflect what's happening
```suggestion
            didSet { // ignore new value if greater than 10
```